miniwallet/contracts/mocks/TestERC721.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+
+/// @custom:security-contact dev@modulo.so
+contract TestERC721 is ERC721, AccessControl {
+    using Counters for Counters.Counter;
+
+    bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
+    Counters.Counter private _tokenIdCounter;
+
+    constructor(uint256 _amount) ERC721("TestERC721", "T721") {
+        _grantRole(DEFAULT_ADMIN_ROLE, msg.sender);
+        _grantRole(MINTER_ROLE, msg.sender);
+        for (uint32 i = 0; i < _amount; i++) {
+            safeMint(msg.sender);
+        }
+    }
+
+    function _baseURI() internal pure override returns (string memory) {
+        return "ipfs://testerc721.modulo.so/";
+    }
+
+    function safeMint(address to) public onlyRole(MINTER_ROLE) {
+        uint256 tokenId = _tokenIdCounter.current();
+        _tokenIdCounter.increment();
+        _safeMint(to, tokenId);
+    }
+
+    // The following functions are overrides required by Solidity.
+
+    function supportsInterface(bytes4 interfaceId)
+        public
+        view
+        override(ERC721, AccessControl)
+        returns (bool)
+    {
+        return super.supportsInterface(interfaceId);
```

Comment:
same issue here

Reply:
same as above

Reply:
Thanks for identifying this @polymorpher 🙏 
[Further research](https://forum.openzeppelin.com/t/derived-contract-must-override-function-supportsinterface/6315/11) confirmed your feedback and I have replaced supportsInterface with code below

```
    function supportsInterface(bytes4 interfaceId)
        public
        view
        override(ERC721, AccessControl)
        returns (bool)
    {
        return (ERC721.supportsInterface(interfaceId) ||
            AccessControl.supportsInterface(interfaceId));
    }
```